### ISSUE TITLE
Gradle module metadata for 2.13.4.1 references non-existent jackson-bom 2.13.4.1

### ISSUE THREAD

**Issue by arjenm** [FasterXML/jackson-databind#3627]
**Describe the bug**

It seems this closed issue re-occured with the release of databind version 2.13.4.1
https://github.com/FasterXML/jackson-databind/issues/3428

This module file references BOM 2.13.4.1 which does not exist.
https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.13.4.1/jackson-databind-2.13.4.1.module

**Version information**
2.13.4 + databind 2.13.4.1

**To Reproduce**

Have a gradle file with:
```
    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.4.1'
```

Which causes this error in Gradle:
```
Could not find com.fasterxml.jackson:jackson-bom:2.13.4.1.
Searched in the following locations:
  - https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
Required by:
    project : > com.fasterxml.jackson.core:jackson-databind:2.13.4.1

```

**Expected behavior**
A working set of jackson packages :)

**Comment by augi** [comment]
Can confirm we can observe this issue as well.

**Comment by cowtowncoder** [comment]
I have a strong sense of Deja Vu here but I thought issue had been fixed: having to do with the way Gradle module metadata plug-in was trying to guess version to fill in.
I think it's this:

https://github.com/FasterXML/jackson-bom/issues/52

but looks like fix is only in 2.14 branch, not 2.13.

The whole Gradle module metadata has been more hassle than worth from maintainer perspective. I just hope there is some value for Gradle users when information has been correct. :-/

It does sound like I need to fix `jackson-bom` 2.13 and try releasing 2.13.4.2 when I find time to do so. Blah.

**Comment by cowtowncoder** [comment]
Ok, published:

* `jackson-databind` 2.13.4.2 with expected `jackson-bom` dependency to 2.13.4
* `jackson-bom` 2.13.4.20221013 to go with above

First one is available via Maven Central, second on its way there. Please LMK if you find further issues.
